project-base/docker/php-fpm/Dockerfile

```diff
@@ -58,22 +50,70 @@ ENV LD_PRELOAD /usr/lib/preloadable_libiconv.so php
 ENV LC_ALL=en_US.utf8 LANG=en_US.utf8 LANGUAGE=en_US.utf8
 
 # overwrite the original entry-point from the PHP Docker image with our own
-COPY docker-php-entrypoint /usr/local/bin/
+COPY ${context_root_path}/docker/php-fpm/docker-php-entrypoint /usr/local/bin/
 RUN chmod +x /usr/local/bin/docker-php-entrypoint
 
 # copy php.ini configuration
-COPY php-ini-overrides.ini /usr/local/etc/php/php.ini
+COPY ${context_root_path}/docker/php-fpm/php-ini-overrides.ini /usr/local/etc/php/php.ini
 
-# allow overwriting UID and GID o the user "www-data" to help solve issues with permissions in mounted volumes
-# if the GID is already in use, we will assign GID 82 instead (82 is the standard uid/gid for "www-data" in Alpine)
-ARG www_data_uid
-ARG www_data_gid
-RUN deluser www-data && (addgroup -g $www_data_gid www-data || addgroup -g 82 www-data) && adduser -u $www_data_uid -D -S -G www-data www-data
-USER www-data
+ARG context_root_path
+
+# install git for computing diffs
+RUN apk add --update git
+
+# install grunt-cli using npm to be able to run grunt watch
+RUN npm install -g grunt-cli
+
+# install Composer
+COPY ${context_root_path}/docker/php-fpm/docker-install-composer /usr/local/bin/docker-install-composer
+RUN chmod +x /usr/local/bin/docker-install-composer && docker-install-composer
 
 # allow configuring the GitHub OAuth token
 ARG github_oauth_token
 RUN if [ -n "$github_oauth_token" ]; then composer config -g github-oauth.github.com $github_oauth_token; fi
 
 # hirak/prestissimo makes the install of Composer dependencies faster by parallel downloading
 RUN composer global require hirak/prestissimo
+
+########################################################################################################################
+
+FROM base as production
+
+ARG context_root_path
+
+COPY --chown=www-data:www-data ${context_root_path}/ /var/www/html
+COPY --chown=www-data:www-data ${context_root_path}/app/config/parameters.yml.dist /var/www/html/app/config/parameters.yml
+COPY --chown=www-data:www-data ${context_root_path}/app/config/domains_urls.yml.dist /var/www/html/app/config/domains_urls.yml
```

Comment:
The lines got removed in the end (in c14811c), because the originally contained wrong path (`home/www/html/`) and the files did not get even used:
https://github.com/shopsys/shopsys/blob/0cbd9e7f6b672bebfe7a7866455ea4ac2b656c78/project-base/docker/php-fpm/Dockerfile-production#L66-L67